src/kwsbin/compute-atwv.cc

```diff
@@ -37,13 +37,31 @@ int main(int argc, char *argv[]) {
 
     const char *usage = "Computes the Actual Term-Weighted Value and prints it."
         "\n"
-        "Usage: compute-atwv [options]  ref-rspecifier hyp-rspecifier [alignment csv]\n"
-        " e.g.: compute-atwv ark:ref.1 ark:hyp.1 ali.csv\n"
+        "Usage: compute-atwv [options] <nof-trials> <ref-rspecifier> <hyp-rspecifier> [alignment-csv]\n"
```

Comment:
need angle brackets around <alignment-csv> ... and is it wspecifier? rspecifier?  wxfilename?